enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -218,9 +239,30 @@ An example IAM Trust Policy:
 
 ```
 
+#### ExternalDNS Operator Details
+
+Prior to this enhancement, the [ExternalDNS Operator API](https://github.com/openshift/external-dns-operator/blob/main/api)
+did not allow users to configure an AWS role ARN field. As a result, there wasn't a supported way to use the External
+DNS Operator to create DNS records in another AWS account within a shared VPC.
+
+The ExternalDNS Operator configures and runs the [ExternalDNS](https://github.com/openshift/external-dns) binary.
+Conveniently, ExternalDNS already supports the [`--aws-assume-role`](https://github.com/openshift/external-dns/blob/fe00b4b83c2263282a9068655e8e3fbbc167b653/docs/faq.md#can-external-dns-manageaddremove-records-in-a-hosted-zone-which-is-setup-in-different-aws-account)
+argument, which uses the specified AWS role ARN when creating new DNS records. Therefore, to support shared VPCs with
+the ExternalDNS Operator, we update the [API](#API-External-DNS-Operator) to expose the `--aws-assume-role` argument
+for the ExternalDNS binary.
+
+By design, ExternalDNS can update both public and private DNS zones using credentials that the cluster-admin provides.
+Therefore, ExternalDNS Operator users are able to use the new [API](#API-External-DNS-Operator) to update both public
+and private DNS zones in other accounts, provided the role ARN has appropriate permissions.
+
+More details on using a role ARN to create DNS records in another AWS account will be documented in ExternalDNS
+Operator's [Docs](https://github.com/openshift/external-dns-operator/tree/main/docs).
+
```

Comment:
We can mention the update of the existing webhook for the aws provider. As you found, it currently forbids the usage of `provider.aws` field without `credentials`.

Reply:
True, I overlooked this. The webhook was just checking for the emptiness of the credentials secret name.